share/templates/CMakeLists.txt

```diff
@@ -17,6 +17,10 @@
 #  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 #=============================================================================
 
+# adding or renaming templates here should also better be done in convert.json, 
+# but definitely need to get reflected in ../instruments/instrumentsxml.h,
```

Comment:
Sorry, but I think the correct grammar should be: `but definitely needs to`? :-)

Reply:
`adding or renamiming templates`, plural here, so IMHO correct there

Reply:
`or`, so that's a single event chosen between two.